assignment/api/src/java/org/sakaiproject/assignment/api/AssignmentService.java

```diff
@@ -817,6 +818,16 @@ public String getDeepLinkWithPermissions(String context, String assignmentId, bo
      */
     public List<MultiGroupRecord> checkSubmissionForUsersInMultipleGroups(String siteId, Group submissionGroup, Collection<Group> asnGroups);
 
+    public boolean correctTime(String timeSheet);
```

Comment:
what Time is correct this method needs a better name

Reply:
Changed to 'timeHasCorrectFormat'